generation/WinSDK/enums.json

```diff
@@ -31932,5 +31957,48 @@
     },
     "members": [],
     "uses": []
+  },
+  {
+    "name": "EMPTY_VOLUME_CACHE_FLAGS",
+    "autoPopulate": {
+      "filter": "EVCF_",
+      "header": "emptyvc.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "inteface": "IEmptyVolumeCache2",
```

Comment:
Why don't these uses show up in the diff?

Reply:
Is it because they are a pointer?
https://docs.microsoft.com/en-us/windows/win32/api/emptyvc/nf-emptyvc-iemptyvolumecache-initialize

Reply:
FOUND IT!!! It was a misspelling of `interface`.

Reply:
Great. @sotteson1 is a schema easy to add to prevent mistakes like this?

Reply:
Issue created at #1226